adapters/Survey/index.js

```diff
@@ -0,0 +1,22 @@
+import baseAdapter from 'adapters/Base';
+
+const SurveyAdapter = () => {
+  const fetchSurveys = (currentPage, authorizationHeader) =>
+    baseAdapter.get(`surveys?page[number]=${currentPage}`, {
+      headers: authorizationHeader,
+    });
+
+  const fetchSurvey = (surveyId, authorizationHeader) =>
+    baseAdapter.get(`surveys/${surveyId}`, {
+      headers: authorizationHeader,
+    });
+
+  return {
+    fetchSurveys,
+    fetchSurvey,
+  };
+};
+
+const surveyAdapter = SurveyAdapter();
+
+export default surveyAdapter;
```

Comment:
Is there a reason for creating the `SurveyAdapter` function that returns 2 other functions, and then exporting the returning value of this `SurveyAdapter()` function?

Would it work by just declaring and exporting both `fetchSurvey(s)` consts?

For example:
```javascript
export const fetchSurvey = [...];
export const fetchSurveys = [...];

export default { fetchSurvey, fetchSurveys };
```

Could be consumed:

```javascript
// either like current
import surveyAdapter from 'adapters/Survey';
// or directly
import { fetchSurveys, fetchSurvey } from 'adapters/Survey';
```

Reply:
I actually followed the [varun-web](https://github.com/nimblehq/varun-web/blob/develop/src/adapters/User/index.ts) implementation here, followed the same pattern throughout the adapters in `adapters/*` folder

Reply:
@carryall do you remember why we did it that way? :D

Reply:
Ahh yeah, got it, in Rails you would export (expose) the whole class. Ok, I'm fine with both, I was just curious to understand why ^^ Thanks!